.circleci/config.yml

```diff
@@ -2,7 +2,8 @@ version: 2
 jobs:
 
   build-caches:
-    machine: true
+    machine:
+      image: ubuntu-2004:202010-01
```

Comment:
this is necessary, since by default it takes some old image with maven 3.2.5 version (where --no-transfer-progress option not available)
This version is recommended in circleci config page